vs2019/rlottie.rc

```diff
@@ -0,0 +1,100 @@
+// Microsoft Visual C++ generated resource script.
+//
+#include "resource.h"
+
+#define APSTUDIO_READONLY_SYMBOLS
+/////////////////////////////////////////////////////////////////////////////
+//
+// Generated from the TEXTINCLUDE 2 resource.
+//
+#include "winres.h"
+
+/////////////////////////////////////////////////////////////////////////////
+#undef APSTUDIO_READONLY_SYMBOLS
+
+/////////////////////////////////////////////////////////////////////////////
+// English (United States) resources
+
+#if !defined(AFX_RESOURCE_DLL) || defined(AFX_TARG_ENU)
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+#pragma code_page(1252)
+
+#ifdef APSTUDIO_INVOKED
+/////////////////////////////////////////////////////////////////////////////
+//
+// TEXTINCLUDE
+//
+
+1 TEXTINCLUDE 
+BEGIN
+    "resource.h\0"
+END
+
+2 TEXTINCLUDE 
+BEGIN
+    "#include ""winres.h""\r\n"
+    "\0"
+END
+
+3 TEXTINCLUDE 
+BEGIN
+    "\r\n"
+    "\0"
+END
+
+#endif    // APSTUDIO_INVOKED
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Version
+//
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION 1,0,0,1
+ PRODUCTVERSION 1,0,0,1
+ FILEFLAGSMASK 0x3fL
+#ifdef _DEBUG
+ FILEFLAGS 0x1L
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS 0x40004L
+ FILETYPE 0x2L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "TechSmith Corporation"
+            VALUE "FileDescription", "rlottie"
+            VALUE "FileVersion", "1.0.0.1"
+            VALUE "InternalName", "rlottie.dll"
+            VALUE "LegalCopyright", "Copyright � 2020 TechSmith Corporation. All rights reserved."
```

Comment:
Actually, is this correct for third-party code? Pango credits Red Hat Software. MP3lame attributes someone else; looks like mp4v2 doesn't mention a company.

Reply:
FYI, we include this version header file, take CSRenderLib.rc as example: https://github.com/TechSmith/Luma/blob/master/CSRenderLib/CSRenderLib.rc#L53

This is how we may use it: https://github.com/TechSmith/Luma/blob/master/CSRenderLib/CSRenderLib.rc#L72

Reply:
Oh, never mind. This is not part of our internal code. What you do is probably okay, just make sure we bump the version each time we have changes.

Reply:
Maybe I'll just leave the company name off